site/content/en/docs/manual/advanced/organization.md

```diff
@@ -144,7 +139,46 @@ by going to **Organization page** > **Leave organization**.
 The organization owner can remove members, by
 clicking on the **Bin** icon.
 
-### Delete organization
+### Inviting members with existing CVAT accounts
+
+To invite members to **Organization** do the following:
+
+1. Go to the [**Organization page**](#organization-page), and click **Invite members**.
+2. Fill in the form (see below).
+
+   ![](/images/image236.jpg)
+
+3. Click **Ok**.
+
+### Inviting members without CVAT accounts
+
+To invite members to **Organization** do the following:
+
+1. Go to the [**Organization page**](#organization-page), and click **Invite members**.
+2. Fill in the form (see below).
+
+   ![](/images/image236.jpg)
+
+3. Click **Ok**.
+4. The person being invited will receive an email with the link.
+
+   ![](/images/invitation_to_org.jpg)
+
+5. Person must click the link and
+   [**set up an account**](/docs/manual/basics/registration/#user-registration).
+
+6. After account is created, the user will be visible
+   on the [**Organization page**](#organization-page)
```

Comment:
It is a bug, invited user sould be visible immidiately. It is fixed in #7073, but that PR needs more improvents to this feature

Reply:
Ok waiting then

Reply:
@mdacoca 
Ok, If we want to merge it before, then lets remove `6` from the list. 
Then somewhere in between 4-5 we can add a line about resending/removing invitations. Thats how it looks in UI:
![image](https://github.com/opencv/cvat/assets/50956430/78b0d757-1f25-4733-abe0-41e3b955fd8d)
You can see with fix from #7073, or admin account on develop.

Basically we can write here that owners/maintainers can resend/remove unaccepted invitations.